storybookjs/storybook
# docs: clarify preset/create-react-app compatibility note

Closes #



## What I did

Fixed a now-outdated statement in a README

Why:

* NPM shows the latest version of this package as 7.x ; the readme said "this" version was 4.x, which is confusing. So we just take out "This".

Questions: 
* should this really go to 'next' branch as github template suggests?

## How to test

- Go to NPM: https://www.npmjs.com/package/@storybook/preset-create-react-app
- Confirm that "this" latest version is no longer 4.x (you should see it as 7.x instead)

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`
